packages/core/src/sass/app/_common/components/account-switcher.scss

```diff
@@ -189,13 +189,23 @@
 
     &__wallets {
         padding: 1.2rem 1.6rem;
+        gap: 0.8rem;
+
+        &-notification-icon {
+            margin-right: 1rem;
+        }
+
+        &-separator {
+            border-right: 1px solid var(--general-section-1);
+            width: 1rem;
+            height: 3.2rem;
+            margin-right: -0.1rem;
```

Comment:
is there any other way to set the margin-right without negative margins?

Reply:
removed unused styles.